### ISSUE TITLE
lr decay

### ISSUE THREAD

**Issue by oye666** [yjn870/RDN-pytorch#6]
param_group['lr'] = args.lr * (0.1 ** (epoch // int(args.num_epochs * 0.8)))

In the paper : The learning rate is initialized to 10−4 for all layers and decreases half for every 200 epochs.

I don't understand what you are doing with the learning rate, can you explain it, thanks a lot
